libhb/common.h

```diff
@@ -85,11 +85,14 @@
 typedef enum
 {
      HB_ERROR_NONE         = 0,
-     HB_ERROR_CANCELED     = 1,
-     HB_ERROR_WRONG_INPUT  = 2,
-     HB_ERROR_INIT         = 3,
-     HB_ERROR_UNKNOWN      = 4,
-     HB_ERROR_READ         = 5
+     HB_ERROR_CANCELED,
+     HB_ERROR_APP_INIT,
+     HB_ERROR_ENC_INIT,
+     HB_ERROR_ENC_INPUT,
+     HB_ERROR_ENC,
+     HB_ERROR_UNKNOWN,
+     HB_ERROR_READ,
+     HB_ERROR_SCAN
 } hb_error_code;
```

Comment:
These error codes can be returned by the CLI (and to some extent, "relied on" by scripts), so we should avoid redefining it too often. Also, I would keep the explicit assignments, makes for a good (or at least quick) reference when we take a look at the error.